docs/unmanaged_configuration.md

```diff
@@ -25,3 +25,39 @@ oc set env dc/fluentd MERGE_JSON_LOG=true
 * Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings
 * Potential buffer storage leak caused by rejected message cycling
 * Overwrite of data for field with same names
+
+## ElasticSearch
+### Exposing elasticsearch service with a route
+```
+# Get ElasticSearch pod name
+ESPOD=$( oc get pods -l component=elasticsearch -o name | sed -e "s/pod\///" )
+
+# Extract CA cert.
+oc exec $ESPOD -- cat /etc/openshift/elasticsearch/secret/admin-ca > ./admin-ca
+
+# Create an yaml file for the ElasticSearch service
```

Comment:
Is it possible to essentially do this same step with `oc expose <servicename>`?

Reply:
I don't see where `oc expose` allows you to specify the certs/keys